src/test/java/no/sikt/nva/scrapers/EmbargoScraperTest.java

```diff
@@ -112,10 +112,12 @@ var record = new Record();
                     "My super secret file.pdf")));
         var updatedRecord = EmbargoParser.checkForEmbargoFromSuppliedEmbargoFile(record, embargos);
         assertThat(appender.getMessages(), not(containsString("Embargo file not found: ")));
-        updatedRecord.getContentBundle()
-            .getContentFiles()
-            .forEach(actualContentFile ->
-                         assertThat(actualContentFile.getEmbargoDate(), is(notNullValue())));
+
+        assertThat(updatedRecord.getContentBundle().getContentFiles(), hasSize(2));
+        assertThat(updatedRecord.getContentBundle().getContentFiles().get(0).getEmbargoDate(),
+                   is(equalTo(Instant.parse("2023-09-30T22:00:00Z"))));
```

Comment:
Gjerne også en test for vinter-tid? Siden det gir annen zulu-tid.

Reply:
Added a test to cover daylight saving when setting embargo date